components/list/demo/list-demo-course-image.js

```diff
@@ -0,0 +1,55 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
```

Comment:
This component just makes a random image to make the demo page look nicer without adding images to the repo.

Reply:
In a couple of instances I used images from course image catalog.  They look nice and there is no worry about license.  ex. [d2l-card](https://github.com/BrightspaceUI/card/blob/master/demo/d2l-card.html).  However, I do not recommend using them for any visual-diff, because the timing can throw off the screenshot, and the delay is ultimately undesirable.

Reply:
Cool, I can use them. Yeah for the visual diff, I just paint the square red with css.